scalableminds/webknossos
# Fixed columns with expandable rows workaround

Antd tables can't handle fixed columns combined with expandable rows.

This PR adds a wrapper component for the antd Table that ensures that fixed columns are only enabled when there are no expanded rows.

This PR also removes the fixed width of colums in the project progress table to ensure that a single project only needs one line.

### URL of deployed dev instance (used for testing):
- https://fixedcloumnsworkaround.webknossos.xyz

### Steps to test:
- Open the dataset table and the task list table.
- Both should have the actions column fixed on the right when there are no expanded rows.
- It should be possible to expand any row without rendering bugs. But keep in mind that this turns (/should turn) off the fixation of the action column. 

### Issues:
- aftermath of #4254 
- [discuss bugreport](https://discuss.webknossos.org/t/actions-menu-for-individual-instances-of-tasks-is-not-visible/1529)
- [project progress bug report](https://discuss.webknossos.org/t/project-progress-tracing-time/1515/8)

------
